src/item_pocket.h

```diff
@@ -399,11 +399,11 @@ class item_pocket
 
         // should the name of this pocket be used as a description
         bool name_as_description = false; // NOLINT(cata-serialize)
+        const pocket_data *data = nullptr; // NOLINT(cata-serialize)
```

Comment:
there's a public get_pocket_data for const accessor

Reply:
Thanks